File: workshop/src/main/java/models/Musee.java
package models;

public class Musee {
    private int id_musee;
    private String nom_musee;
    private String adresse;
    private String ville;
    private int nbr_tickets_disponibles;
    private String description;
    private String image_path;
    public Musee(){

    }
    public Musee(int id_musee, String nom_musee, String adresse, String ville,int nbr_tickets_disponibles,String description, String image_path) {
        this.id_musee = id_musee;
        this.nom_musee = nom_musee;
        this.adresse = adresse;
        this.ville = ville;
        this.nbr_tickets_disponibles=nbr_tickets_disponibles;
        this.description = description;
        this.image_path = image_path;

    }
    public Musee(String nom_musee, String adresse, String ville,int nbr_tickets_disponibles,String description,String image_path) {
        this.nom_musee = nom_musee;
        this.adresse = adresse;
        this.ville = ville;
        this.nbr_tickets_disponibles=nbr_tickets_disponibles;
        this.description = description;
        this.image_path = image_path;

    }
    public Musee(String nom_musee, String adresse, String ville,int nbr_tickets_disponibles,String description) {
        this.nom_musee = nom_musee;
        this.adresse = adresse;
        this.ville = ville;
        this.nbr_tickets_disponibles=nbr_tickets_disponibles;
        this.description = description;

    }
    public int getId_musee(){return id_musee;}
    public String getNom_musee(){return nom_musee;}
    public String getAdresse(){return adresse;}
    public String getVille(){return ville;}
    public int getNbr_tickets_disponibles(){return nbr_tickets_disponibles;}
    public String getDescription(){return description;}

    public String getImage_path() {
        return image_path;
    }

    public void setId_musee(int id_musee) {
        this.id_musee = id_musee;
    }

    public void setNom_musee(String nom_musee) {
        this.nom_musee = nom_musee;
    }
    public void setAdresse(String adresse){this.adresse = adresse;}
    public void setVille(String ville){this.ville = ville;}
    public void setNbr_tickets_disponibles(int nbr_tickets_disponibles){this.nbr_tickets_disponibles=nbr_tickets_disponibles;}
    public void setDescription(String description){this.description = description;}

   public void setImage_path(String image_path){this.image_path=image_path;}
}

File: src/Entity/Musee.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Musee
 *
 * @ORM\Table(name="musee", indexes={@ORM\Index(name="user", columns={"user"})})
 * @ORM\Entity
 */
class Musee
{
    /**
     * @var int
     *
     * @ORM\Column(name="id_musee", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idMusee;

    /**
     * @var string
     *
     * @ORM\Column(name="nom_musee", type="string", length=255, nullable=false)
     */
    private $nomMusee;

    /**
     * @var string
     *
     * @ORM\Column(name="adresse", type="string", length=255, nullable=false)
     */
    private $adresse;

    /**
     * @var string
     *
     * @ORM\Column(name="ville", type="string", length=255, nullable=false)
     */
    private $ville;

    /**
     * @var int
     *
     * @ORM\Column(name="nbr_tickets_disponibles", type="integer", nullable=false)
     */
    private $nbrTicketsDisponibles;

    /**
     * @var string
     *
     * @ORM\Column(name="description", type="string", length=255, nullable=false)
     */
    private $description;

    /**
     * @var \User
     *
     * @ORM\ManyToOne(targetEntity="User")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="user", referencedColumnName="user_id")
     * })
     */
    private $user;

    public function getIdMusee(): ?int
    {
        return $this->idMusee;
    }

    public function getNomMusee(): ?string
    {
        return $this->nomMusee;
    }

    public function setNomMusee(string $nomMusee): self
    {
        $this->nomMusee = $nomMusee;

        return $this;
    }

    public function getAdresse(): ?string
    {
        return $this->adresse;
    }

    public function setAdresse(string $adresse): self
    {
        $this->adresse = $adresse;

        return $this;
    }

    public function getVille(): ?string
    {
        return $this->ville;
    }

    public function setVille(string $ville): self
    {
        $this->ville = $ville;

        return $this;
    }

    public function getNbrTicketsDisponibles(): ?int
    {
        return $this->nbrTicketsDisponibles;
    }

    public function setNbrTicketsDisponibles(int $nbrTicketsDisponibles): self
    {
        $this->nbrTicketsDisponibles = $nbrTicketsDisponibles;

        return $this;
    }

    public function getDescription(): ?string
    {
        return $this->description;
    }

    public function setDescription(string $description): self
    {
        $this->description = $description;

        return $this;
    }

    public function getUser(): ?User
    {
        return $this->user;
    }

    public function setUser(?User $user): self
    {
        $this->user = $user;

        return $this;
    }


}
